src/controllers/customer.ts

```diff
@@ -23,6 +58,7 @@ export class CustomerController {
         }
     }
 
+    // This method never use.
     public async update(request: Request, response: Response) {
```

Comment:
Let's remove it then, not safe to let a customer update the details

Reply:
Removed! ✅